src/Lykke.Cqrs/Middleware/EventLoggingInterceptor.cs

```diff
@@ -1,41 +1,63 @@
 using Lykke.Cqrs.Abstractions.Middleware;
 
 namespace Lykke.Cqrs.Middleware
 {
+    public delegate void EventLoggingDelegate(ILog log, object handlerObject, object @event);
+
+    /// <summary>
+    /// Logging interceptor for events.
+    /// </summary>
     public class EventLoggingInterceptor : IEventInterceptor
     {
         private readonly ILog _log;
-        private readonly Dictionary<Type, Action<ILog, object, object>> _customLoggingActionsMap;
+        private readonly Dictionary<Type, EventLoggingDelegate> _customLoggingActionsMap;
 
+        /// <summary>
+        /// C-tor for old logging.
+        /// </summary>
         [Obsolete]
-        public EventLoggingInterceptor(ILog log, Dictionary<Type, Action<ILog, object, object>> customLoggingActionsMap = null)
+        public EventLoggingInterceptor(ILog log, Dictionary<Type, EventLoggingDelegate> customLoggingActionsMap = null)
         {
             _log = log;
             _customLoggingActionsMap = customLoggingActionsMap;
         }
 
-        public EventLoggingInterceptor(ILogFactory logFactory, Dictionary<Type, Action<ILog, object, object>> customLoggingActionsMap = null)
+        /// <summary>
+        /// C-tor
+        /// </summary>
+        public EventLoggingInterceptor(ILogFactory logFactory, Dictionary<Type, EventLoggingDelegate> customLoggingActionsMap = null)
         {
             _log = logFactory.CreateLog(this);
             _customLoggingActionsMap = customLoggingActionsMap;
         }
 
+        /// <inheritdoc cref="IEventInterceptor"/>
         public Task<CommandHandlingResult> InterceptAsync(IEventInterceptionContext context)
         {
-            var eventType = context.Event.GetType();
-            Action<ILog, object, object> loggingAction = null;
-            if (_customLoggingActionsMap?.TryGetValue(eventType, out loggingAction) ?? false)
+            EventLoggingDelegate loggingAction = null;
+            if (_customLoggingActionsMap?.TryGetValue(context.Event.GetType(), out loggingAction) ?? false)
                 loggingAction?.Invoke(_log, context.HandlerObject, context.Event);
             else
-                _log.WriteInfo(context.HandlerObject.GetType().Name, context.Event?.ToJson(), eventType.Name);
+                DefaultLog(_log, context.HandlerObject, context.Event);
 
             return Task.FromResult(CommandHandlingResult.Ok());
         }
+
+        /// <summary>
+        /// Default logging for events.
+        /// </summary>
+        /// <param name="log">ILog instance.</param>
+        /// <param name="handlerObject">Event handler instance.</param>
+        /// <param name="event">Event.</param>
+        /// <remarks>This could be used in case some custom logging interceptor is implemented for default cases.</remarks>
+        public static void DefaultLog(ILog log, object handlerObject, object @event)
+        {
+            log.WriteInfo(handlerObject.GetType().Name, @event, @event.GetType().Name);
```

Comment:
Type of the event is logged to the ```info```. It's not fit what we have now. Now handler type is ```component```, event type is ```process``` -> this breaks Kibana searches and reports.

Reply:
Yes, we just need to fix reports once after that. Benefits of this change:
1) We don't need to use ...Async method that doesn't do any async work under the hood, but forces us to use await
2) Migration to new Logging becomes easier, cause there are no Info methods where component could be specified.